packages/admin/src/Pages/Page.php

```diff
@@ -184,6 +189,16 @@ protected function getFooterWidgets(): array
         return [];
     }
 
+    protected function getVisibleFooterWidgets(): array
+    {
+        return static::getVisibleWidgets($this->getFooterWidgets());
+    }
+
+    protected static function getVisibleWidgets(array $widgets): array
```

Comment:
```suggestion
    protected function filterVisibleWidgets(array $widgets): array
```

Reply:
Thanks, the `get` naming already felt out of place to me.